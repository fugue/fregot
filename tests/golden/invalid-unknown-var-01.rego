package fregot.tests.invalid_unkown_var_01

test_a {
  b
}
