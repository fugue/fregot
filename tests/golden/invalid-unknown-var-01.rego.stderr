fregot ([34mcompile error[0m):
  "invalid-unknown-var-01.rego" (line 4, column 3):
  unknown variable:

    4|   b
         [31m^[0m

  Undefined variable: b
